iset.mm

```diff
@@ -136192,6 +136204,25 @@ S C_ ( P ( ball ` D ) T ) ) $=
           -> ( abs ` ( ( f ` z ) - y ) ) < e ) ) ) } ) $.
   $}
 
+  ${
+    $d s f x z w $.
+    $( Define the derivative operator on functions on the reals.  This acts on
+       functions to produce a function that is defined where the original
+       function is differentiable, with value the derivative of the function at
+       these points.  The set ` s ` here is the ambient topological space under
+       which we are evaluating the continuity of the difference quotient.
+       Although the definition is valid for any subset of ` CC ` and is
+       well-behaved when ` s ` contains no isolated points, we will restrict
+       our attention to the cases ` s = RR ` or ` s = CC ` for the majority of
+       the development, these corresponding respectively to real and complex
+       differentiation.  (Contributed by Mario Carneiro, 7-Aug-2014.)  (Revised
+       by Jim Kingdon, 25-Jun-2023.) $)
+    df-dvap $a |- _D = ( s e. ~P CC , f e. ( CC ^pm s ) |->
+      U_ x e. ( ( int ` ( ( MetOpen ` ( abs o. - ) ) |`t s ) ) ` dom f )
```

Comment:
Why is ``( ( MetOpen ` ( abs o. - ) ) |`t s )`` used here instead of ``( ( TopOpen ` CCfld ) |`t s )``?

Reply:
We haven't defined `CCfld` in iset.mm yet. Presumably this will happen some day, but a constructive field is not the same as a classical field and I guess I'm kind of undecided/unclear about the whole thing. It seems odd to have to tackle all the field slots if we are only using the metric. I'm also not sure about how to define everything, especially the order (`le`) slot.

The two notations are equivalent in set.mm as seen in https://us.metamath.org/mpeuni/cnfldtopn.html